resources/measures/HPXMLtoOpenStudio/resources/location.rb

```diff
@@ -119,6 +119,20 @@ def self.apply_dst(model, runner, dst_start_date, dst_end_date)
         dst_end_date_day = dst_end_date.split[1].to_i
 
         dst = model.getRunPeriodControlDaylightSavingTime
+
+        if epw_file.daylightSavingStartDate.is_initialized and epw_file.daylightSavingEndDate.is_initialized # HOLIDAYS/DAYLIGHT SAVINGS row in epw file
+          dst_start_date = epw_file.daylightSavingStartDate.get
+          dst_end_date = epw_file.daylightSavingEndDate.get
+          dst_start_date_month = dst_start_date.monthOfYear
+          dst_start_date_day = dst_start_date.dayOfMonth
+          dst_end_date_month = dst_end_date.monthOfYear
+          dst_end_date_day = dst_end_date.dayOfMonth
```

Comment:
Why not simply use `model.getRunPeriod.setUseWeatherFileDaylightSavings(true)`?

Reply:
Actually, if you don't specify the above, it might [default to true](https://bigladdersoftware.com/epx/docs/9-3/input-output-reference/group-location-climate-weather-file-access.html#field-use-weather-file-holidays-and-special-days-1). So simply adding the DST dates to the EPW might be enough?

Reply:
Good questions. I'll check.

Reply:
So, `model.getRunPeriod.getUseWeatherFileDaylightSavings` defaults to true. But then `model.getRunPeriodControlDaylightSavingTime` shows start/end dates different from those in the epw header. Not sure if this is a bug...

Reply:
I found this note in the forward translator for RunPeriod:
```
  // ETH@20121219 - We also do not support pulling the daylight savings period in from the weather
  // file. Explicit RunPeriodControl:DaylightSavingTime objects already override this field, so
  // making this change only affects files without such objects (and likely doesn't change results
  // for those since weather files from the E+ website have no such designations).
```